### DISCUSSION TITLE
How to clean "history" of MPC?

### DISCUSSION THREAD

**Discussion by Spphire** [NVlabs/curobo#85] [Software Q&A]
I'm using curobo's MPC on real robot

As MPC's feature, robot would be stuck in a wierd pose(noted A) sometime, and I will pause MPC tracking(last goal is A) and use other planning method to let robot move to a new pose(noted B), then restart MPC tracking(first goal is B).

At that moment, the robot or MPC will tend to go back to A at first time and then track B. So how to let MPC "forgets" A?

**Comment by balakumar-s** [comment]
This is a useful feature to add. The way to do this would be to have a new function which will reset MPC and start it at a new start joint configuration and a goal pose. 

The only way to do it currently is by recreating an MPC instance from the current joint configuration (this can take 1 second to reinitialize).
